src/templates/app/_layout/head.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Title from '../../_common/components/title.jsx';
+import Favicons from '../../_common/includes/favicons.jsx';
+import AntiClickjack from '../../_common/includes/anti_clickjack.jsx';
+import GoogleOptimizer from '../../_common/includes/google_optimizer.jsx';
```

Comment:
you're adding this file again, head has been moved to common folder. please remove this file and move your changes there.